### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MimiUmoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+* json
+ 
+### How to download
+1. You have the option of downloading the SOM as a zip or git cloning it onto your terminal.
+2. Once you have done that, you can then open the application on a text editor such a vscode or atom to attempt some testing.
+
+### How to use the application
+1.	To use the application, you need to get your own API key for movie reviews. Follow this link https://developer.nytimes.com/  to create your account.
+2.	Once You have your API key, you can use it to set up your base uri and endpoint that is on the lib/services/movie_reviews.rb file. Use this link to help you with the setup https://developer.nytimes.com/docs/movie-reviews-api/1/overview. Always make sure you API key is positioned at the end of your endpoint.
+3.	If you have not got the json and httparty gem installed, make sure you install them by doing  gem install ‘json’ , gem install ‘httparty.’
```
Comment (richgurney) [inline README.md:25]: Could we not create a Gemfile and get the user to run bundle?

Comment (richgurney) [reply README.md:25]: We dont need step 3 if you run bundle

File: spec/movie_reviews_spec.rb [hunk 2]
```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
+
+describe MoviesNYT do
+  context  'Movie Reviews Critics picks' do 
+    before(:all) do 
+      @review_picks = MoviesNYT.new.movie_reviews_nyt
+      @review_picks.reviews('picks')
+    end 
+     
+    it 'Should be an array' do
```
Comment (richgurney) [inline spec/movie_reviews_spec.rb:10]: These test descriptions are really hard to follow, you need to specify the things you expect to be something. For example '   it 'should return the thing as an data_type'

**Review by richgurney — CHANGES_REQUESTED** [review]
Hi Emem, your commit messages need better descriptions. One word messages are not very helpful for anyone looking at what you have done. The code is looking good. Check out the readme there are a few things to improve in there. Make sure you write out a good description like I said below. This is really important for people understanding what you have written.

File: spec/movie_reviews_spec.rb (hunk 2 above)
Comment (MimiUmoh) [reply spec/movie_reviews_spec.rb:10]: Oh okay, thanks for letting me know because I was  kind of confused as to how to write it.
